src/main/java/org/osiam/security/authorization/AccessTokenValidationService.java

```diff
@@ -23,28 +23,44 @@
 
 package org.osiam.security.authorization;
 
-import java.util.*;
+import org.osiam.client.OsiamConnector;
+import org.osiam.client.exception.ConnectionInitializationException;
+import org.osiam.client.oauth.AccessToken;
+import org.osiam.client.oauth.Scope;
+import org.osiam.resources.scim.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.oauth2.common.DefaultOAuth2AccessToken;
+import org.springframework.security.oauth2.common.OAuth2AccessToken;
+import org.springframework.security.oauth2.common.exceptions.InvalidTokenException;
+import org.springframework.security.oauth2.provider.OAuth2Authentication;
+import org.springframework.security.oauth2.provider.OAuth2Request;
+import org.springframework.security.oauth2.provider.token.AccessTokenConverter;
+import org.springframework.security.oauth2.provider.token.ResourceServerTokenServices;
+import org.springframework.stereotype.Service;
 
-import org.osiam.client.*;
-import org.osiam.client.oauth.*;
-import org.osiam.resources.scim.*;
-import org.springframework.beans.factory.annotation.*;
-import org.springframework.security.authentication.*;
-import org.springframework.security.core.*;
-import org.springframework.security.oauth2.common.*;
-import org.springframework.security.oauth2.common.exceptions.*;
-import org.springframework.security.oauth2.provider.*;
-import org.springframework.security.oauth2.provider.token.*;
-import org.springframework.stereotype.*;
+import java.util.*;
 
 @Service
 public class AccessTokenValidationService implements ResourceServerTokenServices {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(AccessTokenValidationService.class);
```

Comment:
Is that an argument for not changing it here? Especially if you create a new logger here. So let's change the "standard" and do it here for the first time even better! BTW: You changed the "standard" in `scim-schema` of [explicit comparison in one test class](https://github.com/osiam/scim-schema/pull/157/files#diff-568079f5fc32727abe1ff6ad9abe4e16L304) but [not everywhere](https://github.com/osiam/scim-schema/blob/332bffbb26e97d04c41d551481a6c2ccfca079df/src/test/groovy/org/osiam/resources/scim/UserJsonSpec.groovy#L112) in the `scim-schema` ;)